boards/arm/lpcxpresso11u68/lpcxpresso11u68-pinctrl.dtsi

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022, NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/lpc11u6x-pinctrl.h>
+
+&pinctrl {
+	pinmux_uart0: pinmux_uart0 {
+		group0 {
+			pinmux = <U0_RXD_PIO1_26>,
+				<U0_TXD_PIO1_27>;
+		};
+	};
+
+	pinmux_uart4: pinmux_uart4 {
+		group0 {
+			pinmux = <U4_RXD_PIO2_11>,
+				<U4_TXD_PIO2_12>;
+		};
+	};
+
+	pinmux_i2c0: pinmux_i2c0 {
+		group0 {
+			pinmux = <I2C0_SCL_PIO0_4>,
+				<I2C0_SDA_PIO0_5>;
+		};
+		nxp,i2c-mode;
+		nxp,i2c-filter = "slow";
```

Comment:
I think this properties should be in the group0 node. Else they are not handled by the `Z_PINCTRL_STATE_PIN_INIT` macro you added in `soc/arm/nxp_lpc/lpc11u6x/pinctrl_soc.h`.

The same comment applies to most of the nodes below.

Reply:
Ditto on above-should be fixed now